packages/strapi-plugin-content-type-builder/admin/src/components/ComponentIconPicker/Wrapper.js

```diff
@@ -2,6 +2,8 @@ import styled from 'styled-components';
 import { colors } from '@buffetjs/styles';
 import PropTypes from 'prop-types';
 
+/* eslint-disable indent */
```

Comment:
Maybe we don't need to disable eslint if we indent with
```
${({ error }) => error && css`
  border: 1px solid ${colors.darkOrange};
  border-radius: 2px;
`}
```

instead of 
```
${({ error }) => {
  if (error) {
    return `
      border: 1px solid ${colors.darkOrange};
      border-radius: 2px;
    `;
   }
   return '';
}}
``` 
which whatever happens is not good

Reply:
I answered in the comment below